content/cpp/concepts/math-functions/terms/isfinite/isfinite.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'isfinite()'
+Description: 'Returns whether the argument is a finite value.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+Returns whether the argument is a finite value.
+
+## Syntax
+
+```cpp
+isfinite(x)
+```
+
```

Comment:
Under this snippet, can we write 1-3 sentences about the domain and range of `x`; what counts as a finite value (and returns `true`)? And what _doesn't_ count as a finite value (and returns `false`)?